ATSQuadroStrategyBase/NinjaTrader 8/bin/Custom/Strategies/AlgoSystemBase.cs

```diff
@@ -2224,7 +2224,7 @@ public virtual StrategyTradeWorkFlowState ProcessWorkFlow(StrategyTradeWorkFlowS
                     //trade per direction test
                     if (Position.MarketPosition == MarketPosition.Long) return ProcessWorkFlow(StrategyTradeWorkFlowState.GoLongValidationRejected);
 
-                    if (IsHistoricalTradeOrPlayBack || IsStrategyUnSafeMode)
+                    if (IsStrategyUnSafeMode)

```

Comment:
This is required -  IsHistoricalTradeOrPlayBack || IsStrategyUnSafeMode  use the same paths a lot of the time-